_partials/self-hosted/_scar-migration.mdx

```diff
@@ -0,0 +1,387 @@
+
+The migration process involves two main steps:
+
+- Pushing the {props.edition} manifests to the OCI registry.
+- Updating the SCAR endpoint to point to the new internal HTTP server.
+
+Once the migration is complete, there is no longer a need to maintain a separate file server exclusively for hosting the
+{props.edition} manifests.
+
+This guide will walk you through the steps required to push the {props.edition} manifests to the OCI registry and update the
+SCAR endpoint.
+
+## Prerequisites
+
+- A deployed self-hosted {props.edition} <PaletteVertexUrlMapper
+    edition={props.edition}
+    text="cluster"
+    palettePath="/install-palette"
+    vertexPath="/install-palette-vertex"
+  /> that uses a customer-managed SCAR to host {props.edition}
+  manifests.
+- Access to the {props.edition} cluster kubeconfig file to verify the SCAR endpoint.
+
+  :::tip
+
+  If you deployed {props.edition} using the Palette CLI, you can download the kubeconfig file from the {props.edition} cluster details
+  page in the system console. Navigate to the **Enterprise Cluster Migration** page and click on the **Admin
+  Kubeconfig** link to download the kubeconfig file. If you deployed {props.edition} to an existing Kubernetes cluster, contact
+  your cluster administrator to obtain the kubeconfig file. For instructions on using the kubeconfig file to access your
+  cluster, refer to the <VersionedLink text="Access Cluster with CLI" url="/clusters/cluster-management/palette-webctl"/>.
+
+  :::
+
+- Access to the file server that hosts the {props.edition} manifests.
+- The {props.edition} cluster must have been upgraded to version `4.5.13` or later. This is required for the SCAR migration to
+  function properly.
+- Access to the {props.edition} system console.
+- Ensure the following software is installed and available in the environment hosting the file server. For example, if
+  you deployed an airgapped instance of {props.edition} to VMware using an <PaletteVertexUrlMapper
+    edition={props.edition}
+    text="OVA"
+    palettePath="/install-palette/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions"
+    vertexPath="/install-palette-vertex/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions"
+  />, these tools must be available on your airgap support VM.
+
+  - [tar](https://www.gnu.org/software/tar/)
+  - [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) - Required only for AWS
+    ECR.
+  - [ORAS](https://oras.land/docs/installation/) v1.0.0
+
+  :::warning
+
+  This specific version of ORAS is explicitly required for pushing packs to OCI registries.
+
+  :::
+
+- Ensure the following software is installed and available locally on your workstation. 
+  - [curl](https://curl.se/docs/install.html)
+  - [jq](https://jqlang.github.io/jq/download/)
+  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+
+## Migrate SCAR
+
```

Comment:
@caroldelwing - All step looks good. We have added step 1 for migration [wiki](https://spectrocloud.atlassian.net/wiki/spaces/ENGINEERIN/pages/2654273537/Scar+to+OCI+Registry+Migration#Scar-to-OCI-Migration) to make sure we clean earlier secrets (if any)

Please add to the docs

Reply:
Done :)